test/test_torch.py

```diff
@@ -7072,10 +7072,7 @@ def tmp(dtype, device):
         _cpu_types, True, [tf32_on_and_off(0.05), _wrap_warn_once("This overload of baddbmm_? is deprecated")]),
     ('bmm', '', _small_3d, lambda t, d: [_small_3d(t, d)],
         1e-5, 1e-5, 1e-5, _float_types_no_half, _cpu_types, False),
-    ('addcdiv', '', _small_2d,
```

Comment:
Thanks, because of this, I just recollected that I'd have to remove these entries for some other ops!